src/frontends/common/src/manager.cpp

```diff
@@ -79,6 +79,7 @@ class FrontEndManager::Impl {
         std::lock_guard<std::mutex> guard(m_loading_mutex);
         for (auto& plugin_info : m_plugins) {
             if (!plugin_info.load()) {
+                std::cerr << "Frontend load failed: " << plugin_info.m_file_path << "\n";
```

Comment:
do we need it?

Reply:
I think this is nice to have I had a situation where it helped. I changed `cerr` with `OPENVINO_DEBUG` so it looks better